.claude/commands/gh-issue-chore-deps.md

```
@@ -0,0 +1,182 @@
+  exit 1
+fi
+
+# Hardcoded repository info for binbashar/le-tf-infra-aws
+OWNER="binbashar"
+REPO="le-tf-infra-aws"
+
+# Auto-detect if it's an issue or PR using GitHub API
+echo "🔍 Detecting if #$ISSUE_NUMBER is an issue or PR..."
+if gh pr view "$ISSUE_NUMBER" --repo "$OWNER/$REPO" &> /dev/null; then
+  ISSUE_TYPE="pull"
+  ISSUE_URL="https://github.com/$OWNER/$REPO/pull/$ISSUE_NUMBER"
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)